Automattic/wp-calypso
# Assembler: Update zoom control to be a button instead of a slider

Related to #85123

## Proposed Changes

This PR updates the zoom control from a slider to a button. Clicking the button will zoom out the device viewport by 50%. 

| Before | After |
| --- | --- | 
| ![Screenshot 2023-12-27 at 12 04 23 PM](https://github.com/Automattic/wp-calypso/assets/797888/c54ae269-fa7b-488f-bcc4-c260bfb76bf6) | ![Screenshot 2023-12-27 at 12 04 51 PM](https://github.com/Automattic/wp-calypso/assets/797888/c438b8fd-40d0-4ac0-99cc-2b2eb7f31dca) |

Final version:

https://github.com/Automattic/wp-calypso/assets/797888/9f4d96b8-f8b7-4ad8-9092-5573a0fc561a

## Testing Instructions



* Head to the Assembler.
* Select any pattern.
* Ensure that the zoom toggle is visible.
* Ensure that the zoom toggle works as described above.

## Pre-merge Checklist
